src/python/env.py

```diff
@@ -247,7 +267,15 @@ def step(  # pyright: ignore[reportIncompatibleMethodOverride]
         # Frameskip
         reward = 0.0
         for _ in range(frameskip):
-            reward += self.ale.act(self._action_set[action])
+            if self.continuous:
+                action = tuple(action)
```

Comment:
We can do `r, theta, fire = action.tolist()` here.

Reply:
done